Doc/whatsnew/3.11.rst

```diff
@@ -1173,6 +1173,61 @@ Deprecated
   (Contributed by Serhiy Storchaka in :gh:`91760`.)
 
 
+Pending Removal in Python 3.12
+==============================
+
+The following APIs have been deprecated in earlier Python releases, and are now
+pending removal in Python 3.12.
+
+Python API:
+
+* :class:`pkgutil.ImpImporter`
+* :class:`pkgutil.ImpLoader`
+* :envvar:`PYTHONTHREADDEBUG`
+* :func:`importlib.find_loader`
+* :func:`importlib.util.module_for_loader`
+* :func:`importlib.util.set_loader_wrapper`
+* :func:`importlib.util.set_package_wrapper`
+* :meth:`importlib.abc.Loader.module_repr`
+* :meth:`importlib.abc.Loadermodule_repr`
+* :meth:`importlib.abc.MetaPathFinder.find_module`
+* :meth:`importlib.abc.MetaPathFinder.find_module`
+* :meth:`importlib.abc.PathEntryFinder.find_loader`
+* :meth:`importlib.abc.PathEntryFinder.find_module`
+* :meth:`importlib.machinery.BuiltinImporter.find_module`
+* :meth:`importlib.machinery.BuiltinLoader.module_repr`
+* :meth:`importlib.machinery.FileFinder.find_loader`
+* :meth:`importlib.machinery.FileFinder.find_module`
+* :meth:`importlib.machinery.FrozenImporter.find_module`
+* :meth:`importlib.machinery.FrozenLoader.module_repr`
+* :meth:`importlib.machinery.PathFinder.find_module`
+* :meth:`importlib.machinery.WindowsRegistryFinder.find_module`
+* :meth:`pathlib.Path.link_to`
+* The entire :ref:`distutils namespace <distutils-deprecated>`
+* ``cgi.log()``
+* ``sqlite3.OptimizedUnicode``
+* ``sqlite3.enable_shared_cache``
+
+C API:
```

Comment:
> If you click on a function, you can find the deprecation note there and usually you get a suggestion to replace the deprecated function. For example, importlib.find_loader() suggests using importlib.util.find_spec() instead.

Yeah; typically, I'd think if a user actually uses a function, they'd want more details anyway, and if functions don't have clear guidance on why they are deprecated and what to replace them with (or not), then it can be added there.

Reply:
Ola meu amiga